### DISCUSSION TITLE
luigi and PWA

### DISCUSSION THREAD

**Discussion by mehdijouan** [SAP/luigi#3281] [Q&A] [answered]
Hello,
Before starting to use Luigi, I would like to know if it can be used as a [progressive web app](https://web.dev/progressive-web-apps/) ?
I would like to use it to create an installable web application that must be able to work even without network.

Thanks in advance for your guidance.
Mehdi.

**Comment by hardl** [comment]
Hi Mehdi,
since luigi core and client libs only contain static assets and all application logic is done only client-side, i don't see a reason, why it should not work.

Best,
Philipp

**Comment by hardl** [comment, marked as answer]
I did a quick POC, you can find the sources here: https://github.com/hardl/luigi-pwa-example, and the app running on pages here: https://hardl.github.io/luigi-pwa-example

**Reply by mehdijouan** [reply]
Thanks for the example, It'll be helpful!
